api/v1alpha1/pulp_types.go

```diff
@@ -453,47 +453,39 @@ type Database struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	ExternalDB ExternalDB `json:"external_db,omitempty"`
 
-	// PostgreSQL version
-	// +kubebuilder:default:="13"
+	// PostgreSQL version [default: "13"]
```

Comment:
We are handling the default values, if nothing defined, in the controller already.
I'm adding the [default: ...] for documentation purposes.

https://github.com/pulp/pulp-operator/blob/go-1.0.0-beta/controllers/pulp/database.go#L195-L205